### PR TITLE
chore: prepare the release of v9.2

### PR DESCRIPTION
**Pull request by kisvegabor**
**Do not squash**

### Notes
- Update the [Documentation](https://github.com/lvgl/lvgl/tree/master/docs) if needed.
- Add [Examples](https://github.com/lvgl/lvgl/tree/master/examples) if relevant.
- Add [Tests](https://github.com/lvgl/lvgl/blob/master/tests/README.md) if applicable.
- If you added new options to `lv_conf_template.h` run [lv_conf_internal_gen.py](https://github.com/lvgl/lvgl/blob/master/scripts/lv_conf_internal_gen.py) and update [Kconfig](https://github.com/lvgl/lvgl/blob/master/Kconfig).
- Run `scripts/code-format.py` (`astyle v3.4.12` needs to installed by running `cd scripts; ./install_astyle.sh`) and follow the [Code Conventions](https://docs.lvgl.io/master/CODING_STYLE.html).
- Mark the Pull request as [Draft](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-stage-of-a-pull-request) while you are working on the first version, and mark is as _Ready_ when it's ready for review.
- When changes were requested, [re-request review](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review) to notify the maintainers.
- Help us to review this Pull Request! Anyone can [approve or request changes](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/approving-a-pull-request-with-required-reviews).

### CODE REVIEW

File: docs/CHANGELOG.rst [hunk 1]
```diff
@@ -1,321 +1,484 @@
...
+- **chore(vg_lite): fix typo** `5993 <https://github.com/lvgl/lvgl/pull/5993>`__
+- **chore(vg_lite): remove duplicate parameter judgments** `5960 <https://github.com/lvgl/lvgl/pull/5960>`__
+- **chore(PR): modify the astyle version description in PR** `5963 <https://github.com/lvgl/lvgl/pull/5963>`__
+- **chore(cmsis-pack): minor update** `5948 <https://github.com/lvgl/lvgl/pull/5948>`__
+- **chore(observer): rename lv_button_bind_checked to lv_obj_bind_checked** `5944 <https://github.com/lvgl/lvgl/pull/5944>`__
+- **refactor(font): refactor font glyph data acquire method** `5884 <https://github.com/lvgl/lvgl/pull/5884>`__
+- **Optimize lv_color_16_16_mix() when c1 == c2** `5929 <https://github.com/lvgl/lvgl/pull/5929>`__
+- **chore(cache): adjust includes** `5921 <https://github.com/lvgl/lvgl/pull/5921>`__
+- **refacter(conf): use defines for standard includes** `5767 <https://github.com/lvgl/lvgl/pull/5767>`__
+- **refactor(image_decoder): refactor image decoder and image cache** `5890 <https://github.com/lvgl/lvgl/pull/5890>`__
```
Comment (liamHowatt) [inline docs/CHANGELOG.rst]:
```suggestion
```

File: docs/CHANGELOG.rst [hunk 2]
```diff
@@ -1,321 +1,484 @@
...
-`v9.1.0 <https://github.com/lvgl/lvgl/compare/v9.0.0...v9.1.0>`__ 20 March 2024
-----------------------------------------------------------------------------------------------
+And many smaller fixes and features
+
+Breaking Changes
+~~~~~~~~~~~~~~~~
+
+``lvgl_private.h`` was introduced to better separate internal and public APIs and data. Your application might rely on APIs or data that are now available via ``lvgl_private.h``.
+
+The resolves it either
```
Comment (liamHowatt) [inline docs/CHANGELOG.rst]:
```suggestion
Resolve it by either
```

File: docs/CHANGELOG.rst [hunk 3]
```diff
@@ -1,321 +1,484 @@
-.. _changelog:
+`v9.2 <https://github.com/lvgl/lvgl/compare/v9.1.0...v9.2>`__ 26 August 2024
```
Comment (liamHowatt) [inline docs/CHANGELOG.rst]: Make sure the tag actually gets named "v9.2" and not "v9.2.0", otherwise, change this line to say "v9.2.0"

**Review by liamHowatt — COMMENTED** [review]
Should the doc comment at the top of `env_support/cmsis-pack/lv_conf_cmsis.h` be updated?

**Comment by kisvegabor** [comment]
All fixed, thank you! 

The last piece is the CMSIS update from @GorgonMeducer.

**Pull request merged**